scipy/signal/filter_design.py

```diff
@@ -3504,6 +3504,196 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, ab=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    Parameters
+    ----------
+    w0 : float
+        A scalar that must satisfy  0 < w0 < 1. For notch filters,
+        w0 is the normalized frequency to remove from the filtered
+        signal (w0 = 1 corresponds to half of the sample frequency).
+    bw : float
+        Filter bandwidth.
+    ab : float, optional
+        Bandwidht maximum atenuation in decibels. By default, `ab`
```

Comment:
Typos in "Bandwidht" and "atenuation".

I'm not really an expert, but "Bandwidht maximum atenuation in decibels" doesn't sound very clear. Is it written in some book? I would write it like "Attenuation in decibels at bandwidth edge frequencies" (it's just a guess, better to copy a phrase from some book).